Carting/src/WebApi/Controllers/V2/CartController.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Carting.WebApi.Application.Carts.Commands.AddItemToCart;
+using Carting.WebApi.Application.Carts.Commands.RemoveItemFromCartCommand;
+using Carting.WebApi.Application.Carts.Queries.GetCart;
+using Carting.WebApi.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Carting.WebApi.Controllers.V2;
+
+[ApiVersion("2")]
+public class CartsController : ApiControllerBase
+{
+    /// <summary>
+    /// Returns a specific cart and its associated items
+    /// </summary>
+    /// <param name="query"></param>
+    /// <returns></returns>
+    /// <remarks>
+    /// Sample request:
+    ///
+    ///     GET /
+    ///     {
+    ///        "CartId": 1
+    ///     }
```

Comment:
Unchanged sample response?
Writing remarks is painful :) as you have to maintain it

Reply:
Yes, you're correct.. :) Fixed it